### ISSUE TITLE
Essentials recipes

### ISSUE THREAD

**Issue by wooorm** [tmcw/flair#37]
A good addition may be to describe how to do essentials:

- How to make simple / honey / chamomile / etc syrup
- How to make a perfect orange twist

**Comment by wooorm** [comment]
Different but related idea: we could add some content on topics such as all the types of glassware, or the different measurements, or even ingredients (like describing the different rums)

**Comment by wooorm** [comment]
I would love to have a page on fernet branca or such, with some history, a list of recipes you can make with it, neighboring ingredients, a minimal grid of only the cocktails made with it?!

**Comment by tmcw** [comment]
Absolutely! Would be super feasible, main challenge is getting the content right. I have no idea how to make a perfect orange twist 😛

**Comment by wooorm** [comment]
Some for the syrups: https://imbibemagazine.com/how-to-make-simple-syrups/
